srcs/server/Connection.cpp

```diff
@@ -39,11 +39,14 @@ SendResponseStatus::Type	Connection::SendResponse() {
 		response_ = response_factory_->Response();
 		raw_response_ = response_->Content();
 	}
-	ssize_t nbytes = send(socket_, raw_response_.c_str(), raw_response_.size(), 0);
-	if (nbytes <= 0) {
-		return SendResponseStatus::kFail;
+	if (!response_->IsCgi()) {
```

Comment:
I understand you make this to send CGI headers through the cgi handler rather than here, right?

Reply:
Yes, that's it :+1: